src/components/Search/SearchResults.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext, FC } from "react";
+import useSongList from "../../hooks/useSongList";
+import { BrowserSessionContext } from "../../BrowserSessionProvider";
+import SelectInput from "ink-select-input/build";
+import { Text, useFocusManager, useInput } from "ink";
+import Gradient from 'ink-gradient';
+import Spinner from 'ink-spinner';
+
+type SRState = {
+  searchResultActive: boolean,
+  setSearchResultActive: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const SearchResults:FC<{state: SRState}> = ({state}) => {
+
+  const session = useContext(BrowserSessionContext);
+  const songList = useSongList();
+
+  const { focus } = useFocusManager();
+
+  useInput ((_, key) => {
+    if (!state.searchResultActive) return;
+
+    if (key.leftArrow) {
+      state.setSearchResultActive(false);
+      focus('search-bar');
+    }
+    if (key.rightArrow) {
+      // do nothing for now.
+    }
+  })
+
+  const handleSelect = (selection) => {
+    state.setSearchResultActive(false);
+    focus('playback-controls');
```

Comment:
another hardcoded focus

Reply:
Hardcoded to return to playback controls once choice has been made. This one was intentional.